src/configLoaders/DefaultConfigLoader.ts

```diff
@@ -0,0 +1,20 @@
+import Configloader from './ConfigLoader';
+import * as path from 'path';
+
+export default class DefaultConfigEditor implements Configloader {
+  private fs: any;
+  private projectRoot: string;
+
+  constructor(projectRoot: string, fs: any) {
+    this.projectRoot = projectRoot;
+    this.fs = fs;
+  }
+
+  public loadConfig(): string {
+    return JSON.stringify(this.getJestConfigFromPackageJson(this.projectRoot));
+  }
+
+  private getJestConfigFromPackageJson(projectRoot: string) {
```

Comment:
Does this work with the [Multi-Project-Runner & Configuration Overhaul](http://facebook.github.io/jest/blog/2017/05/06/jest-20-delightful-testing-multi-project-runner.html) in Jest 20?

Reply:
Well I did see an option to add multiple projectroots (as an array) to the run function of Jest. Currently I programmed it so that only one projectroot is provided in that array. Is there a case where we want to test multiple projects at the same time with Stryker?

Reply:
I don't really think we need it immediately, as long as we have an option to add / extend it later on.

Reply:
That should be possible, we could add a `projectroot` property in the future which will accept a string or array of strings. I am not sure on how this will work with the jest configuration in the `package.json` though. But since the configuration from the jest config is loaded this shouldn't be a problem™